samples/sensor/thermometer/boards/mimxrt1180_evk_mimxrt1189_cm33.overlay

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* The p3t1755 shield board is plugged into the arduino interface on
+ * the MIMXRT1180-EVK board, and the J13 on the shield board is connected
+ * to the J50 on the RT1180 referred below.
+ * J13-1 -> J50-1; J13-2 -> J50-2;
+ * J13-3 -> J50-3; J13-4 -> J50-4;
+ */
+
+#include <freq.h>
+
+/ {
+	aliases {
+		ambient-temp0 = &p3t1755;
+	};
+};
+
+&i3c2 {
+	status = "okay";
+
+	i2c-scl-hz = <DT_FREQ_K(400)>;
+	i3c-scl-hz = <DT_FREQ_M(4)>;
+	i3c-od-scl-hz = <DT_FREQ_K(1500)>;
+
+	p3t1755: p3t1755@4800000236152a0090 {
```

Comment:
Just a question- should we be defining this as a shield? It seems like we might reuse this sensor for I3C enablement on a bunch of platforms, right?

Reply:
1. The name "shield" wasn't defined by me. NXP release the P3T1755DP Arduino® Shield Evaluation Board officially on [NXP web](https://www.nxp.com/design/design-center/development-boards-and-designs/analog-toolbox/arduino-shields-solutions/p3t1755dp-arduino-shield-evaluation-board:P3T1755DP-ARD)
2. Yes, we can use this board on a bunch of platforms, if exist arduino interface and standalone I3C interface on nxp board.

Reply:
Makes sense. We don't need to do this as part of this PR, but we should consider defining this as a shield to simplify future enablement- especially because it is using the Arduino interface. Is the I3C bus on the RT1180 exposed on the same pins Arduino typically uses for I2C?

Reply:
Thank you for pointing out. I have added p3t1755dp_ard shield board and support shield board on RT1180 board.